### PR TITLE
Add Tooltip for JDK Distros Explanation

### PR DESCRIPTION
**Pull request by hjungwoo01**
This PR is connected to PR #50 and aims to add a bit more context to what “JDK distros” means.

### CODE REVIEW

File: tutorials/javaInstallationMac.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This guide explains how to install the **==[Java 17 JDK+FX Azul distribution](ht
 
 **Why this specific version of JDK?**
 
-* This version comes with <tooltip content="a Java framework used to develop Java GUI programs">JavaFX</tooltip> (which is not included in most JDK distros).
+* This version comes with <tooltip content="a Java framework used to develop Java GUI programs">JavaFX</tooltip> (which is not included in most <tooltip content="different versions or implementations of the Java Development Kit (JDK), for example, Oracle JDK and OpenJDK">JDK distros</tooltip>).
```
Comment (damithc) [inline tutorials/javaInstallationMac.md:13]:
```suggestion
* This version comes with <tooltip content="a Java framework used to develop Java GUI programs">JavaFX</tooltip> (which is not included in most <popover content="distros' is a shorter term for 'distributions'. So, 'JDK distros' are different versions or implementations of the Java Development Kit (JDK) e.g., Oracle JDK, OpenJDK, Azul JDKs etc.">JDK distros</popover>).
```
As the text is longer, better to use a popover instead (similar to tooltip but more optimized for longer text)

**Review by baskargopinath — APPROVED** [review]
LGTM

**Review by jedkohjk — APPROVED** [review]
LGTM!

File: tutorials/javaInstallationMac.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ This guide explains how to install the **==[Java 17 JDK+FX Azul distribution](ht
 
 **Why this specific version of JDK?**
 
-* This version comes with <tooltip content="a Java framework used to develop Java GUI programs">JavaFX</tooltip> (which is not included in most JDK distros).
+* This version comes with <tooltip content="a Java framework used to develop Java GUI programs">JavaFX</tooltip> (which is not included in most <popover content="distros' is a shorter term for 'distributions'. So, 'JDK distros' are different versions or implementations of the Java Development Kit (JDK) e.g., Oracle JDK, OpenJDK, Azul JDKs etc.">JDK distros</popover>).
```
Comment (NeoHW) [inline tutorials/javaInstallationMac.md:13]: There might be a typo in the first `distros'`, do you mean for it to be either `distros` or `'distros'`?

**Review by NeoHW — COMMENTED** [review]
good informative change!

**Pull request merged**